README.md

```diff
@@ -334,14 +350,20 @@ defradb client schema add --url localhost:9182 '
 '
 ```
 
-Set *nodeA* to actively replicate the "Article" collection to *nodeB*:
+Then copy the peer info from *nodeB*:
 
 ```shell
-defradb client rpc replicator set -c "Article" /ip4/0.0.0.0/tcp/9172/p2p/<peerID_of_nodeB>
+defradb client p2p info
 ```
 
-As we add or update documents in the "Article" collection on *nodeA*, they will be actively pushed to *nodeB*. Note that changes to *nodeB* will still be passively published back to *nodeA*, via pubsub.
+Set *nodeA* to actively replicate the Article collection to *nodeB*:
+
+```shell
+defradb client p2p replicator set -c Article <peer_info_of_nodeB>
```

Comment:
suggestion: I had to check the help text of `defradb client p2p replicator set` to be sure that `<peer_info_of_nodeB>` meant the full json object - it might be worth renaming this to reflect that.

Reply:
I updated it to `nodeB_peer_info_json` to hopefully make it more obvious.